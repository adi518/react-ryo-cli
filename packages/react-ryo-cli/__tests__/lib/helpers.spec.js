const {
  getLibraryName,
  getCracoCliCommandCreator
} = require("../../lib/helpers.js");

console.log = () => {}; // eslint-disable-line
console.error = () => {}; // eslint-disable-line

describe("getLibraryName", () => {
  it("should return in pascal case", () => {
    expect(getLibraryName({ name: "foo-bar" })).toMatchSnapshot();
  });
});

describe("getCracoCliCommandCreator", () => {
  it("should add Craco arguments", () => {
    const createCracoCliCommand = getCracoCliCommandCreator();
    expect(createCracoCliCommand("foo")).toMatchSnapshot();
  });
});
