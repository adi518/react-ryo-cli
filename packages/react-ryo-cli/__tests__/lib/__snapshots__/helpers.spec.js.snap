// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCracoCliCommandCreator should add Craco arguments 1`] = `
Array [
  "D:\\\\dev\\\\@react-ryo-cli\\\\node_modules\\\\@craco\\\\craco\\\\bin\\\\craco.js",
  "foo",
  "--config",
  "D:\\\\dev\\\\@react-ryo-cli\\\\packages\\\\react-ryo-cli\\\\config\\\\craco\\\\craco.config.js",
]
`;

exports[`getLibraryName should return in pascal case 1`] = `"FooBar"`;
